NineChronicles.Headless/GraphTypes/StandaloneMutation.cs

```diff
@@ -154,17 +155,10 @@ IConfiguration configuration
                         Description = "A string value of the value to be transferred.",
                         Name = "amount",
                     },
-                    new QueryArgument<NonNullGraphType<LongGraphType>>
+                    new QueryArgument<NonNullGraphType<CurrencyEnumType>>
                     {
-                        Description = "A sender's transaction counter. You can get it through nextTxNonce().",
-                        Name = "txNonce",
-                    },
```

Comment:
I remembered it was designed to restrict mistake to send multiple times. cc @longfin